sites/footers/includes/footer.html

```diff
@@ -1,5 +1,5 @@
 <footer id="wb-info">
-{% unless page.nofooter %}
+{% unless page.nofooterAbout %}
```

Comment:
Fix the camel case here and on the other property below otherwise your changes do not appear for all of the working examples.
```suggestion
{% unless page.noFooterAbout %}
```